src/features/content-explorer/item-list/ItemList.js

```diff
@@ -188,8 +190,30 @@ const ItemList = ({
         };
     }
 
+    const addAutoSizer = WrappedComponent => {
+        return props => {
+            return isResponsive ? (
+                <div style={{ flex: 1 }}>
+                    <AutoSizer>
+                        {({ width: w, height: h }) => <WrappedComponent {...props} width={w} height={h} />}
+                    </AutoSizer>
+                </div>
+            ) : (
+                <WrappedComponent {...props} />
+            );
+        };
+    };
+
+    TableComponent = addAutoSizer(TableComponent);
+
     return (
-        <div className={classNames('content-explorer-item-list table', className)}>
+        <div
+            className={classNames(
+                'content-explorer-item-list table',
+                isResponsive ? 'content-explorer-item-list--isResponsive' : '',
+                className,
+            )}
```

Comment:
```suggestion
            className={classNames('content-explorer-item-list table', className, {
                'bdl-ItemList--responsive': isResponsive,
            })}
```

I'm not 100% sure about the SUIT name here but seems correct?